patches/linux_uio/uio_pci_dma.c

```diff
@@ -368,7 +368,7 @@ probe
 
         if(pci_device->resource[i].flags & IORESOURCE_MEM)
         {
-            dma_device->attr_bar[i].attr.name = kmalloc(10*sizeof(char), GFP_KERNEL);
+            dma_device->attr_bar[i].attr.name = kzalloc(10*sizeof(char), GFP_KERNEL);
```

Comment:
Does this initialization improve anything? If the string is not too long, sprintf will properly zero-terminate it.

Reply:
I initialized this mostly with debugging in mind. Can be reverted. It should not make a functional difference.